cmd/vicadmin/vicadm.go

```diff
@@ -544,27 +575,26 @@ func (s *server) serve() error {
 	s.handleFunc("/container-logs.tar.gz", s.tarContainerLogs)
```

Comment:
Why don't have authenticate for log download?
And just curious, is there any way to authenticate all request to vicadmin service? For example, to a base URL, and then any child URL is applied dynamically. I'm not familiar with Go solution, so just confirm for it.